Mathlib/Algebra/BigOperators/Multiset/Basic.lean

```diff
@@ -406,6 +406,16 @@ theorem prod_map_le_prod_map {s : Multiset ι} (f : ι → α) (g : ι → α) (
 #align multiset.prod_map_le_prod_map Multiset.prod_map_le_prod_map
 #align multiset.sum_map_le_sum_map Multiset.sum_map_le_sum_map
 
+@[to_additive]
+theorem prod_map_lt_prod_map {s : Multiset ι} {f g : ι → α} [CovariantClass α α (· * ·) (· < ·)]
+    (all_le : ∀ i ∈ s, f i ≤ g i) (exists_lt : ∃ i ∈ s, f i < g i) :
+    (s.map f).prod < (s.map g).prod := by
+  rcases s with ⟨l⟩
+  simp only [quot_mk_to_coe'', coe_map, coe_prod]
+  apply List.prod_lt_prod'
+  · exact all_le
+  · exact exists_lt
```

Comment:
Can you collapse this into a single term?

Reply:
Yes.